server.js

```diff
@@ -3476,16 +3476,20 @@ cache(function(query_data, match, sendBadge, request) {
   var repo = match[4];
   var branch = match[5] || 'master';
   var format = match[6];
-  var apiUrl = 'https://raw.githubusercontent.com/' + user + '/' + repo + '/' + branch + '/' + type + '.json';
+  var hasTokens = !!githubAuth.getTokenDebugInfo({ sanitize: false }).tokens.length;
```

Comment:
Hey, thanks for updating this. This method call is intended for generating debug information and it would be surprising to see it used here. It's also not the right condition. Production uses token rotation, so this will be true, and that's where we want to avoid sending auth information.

I think the thing to check is whether a _static_ token is configured: whether either `serverSecrets.gh_token` or `serverSecrets.gh_client_id` is set.